tests/credential_plugins_test.py

```diff
@@ -137,13 +137,19 @@ class TestDelineaImports:
     library, so these tests are designed to fail if these wind up using the
     fallback import."""
 
-    def test_dsv_import(self):
-        from awx_plugins.credentials.dsv import SecretsVault  # noqa
+    def test_dsv_import(self) -> None:
```

Comment:
This bit belongs in #26. But that's okay: it'll disappear when you rebase this PR.

Reply:
So now you need to resolve a conflict when rebasing.

Reply:
I did that. Marking as resolved.